### PR TITLE
chore: run yarn build to generate sn-table-ext

### PR DESCRIPTION
**Pull request by LiKang6688**
miss sn-table-ext in the packing
https://unpkg.com/browse/@nebula.js/sn-table@1.9.0/

add `module": "core/esm/index.js"` to fix Error: "export 'useId' (imported as 'React') was not found in 'react'" after upgrading" (https://github.com/mui/material-ui/issues/29860)

### CODE REVIEW

**Comment by haxxmaxx** [comment]
we need this included in the package right? So we need to include the sense script somewhere where we build?

**Comment by LiKang6688** [comment]
> we need this included in the package right? So we need to include the sense script somewhere where we build?

Yes, correct. I will do that. Thanks

**Comment by haxxmaxx** [comment]
@Caele would be great if you can have a look so we get the bundling correct once and for all :)

File: package.json [hunk 1]
```diff
@@ -22,19 +22,22 @@
...
     "api-specifications",
-    "core"
+    "core",
+    "sn-table-ext"
   ],
   "engines": {
     "node": ">=16"
   },
   "main": "dist/sn-table.js",
+  "module": "core/esm/index.js",
```
Comment (LiKang6688) [inline package.json:32]: To fix error: "export 'useId' (imported as 'React') was not found in 'react'" after upgrading" (https://github.com/mui/material-ui/issues/29860)

**Review by Caele — APPROVED** [review]
Looks correct, but it is the published output that matters.

**Pull request merged**